src/js/poster-image.js

```diff
@@ -131,7 +138,7 @@ class PosterImage extends ClickableComponent {
    *        The URL to the source for the `PosterImage`.
    */
   setSrc(url) {
-    this.el_.querySelector('img').src = url;
+    this.$('img').src = url;
```

Comment:
this will throw if someone tries to set a src and the img element isn't present.

Reply:
I was thinking a poster is only set with `player.poster('...')`, but you're right of course; `setSrc()` isn't "private" so does need to work independently of `update()`.